multi-engine/README.md

```diff
@@ -0,0 +1,83 @@
+# MultiEngine on DJL
+This is an example of how to run multiple engines in one Java process using DJL.
+
+Why is it good to be able to run multiple engines in one Java process?
+
+Many deep learning frameworks have their individual strength and weaknesses when doing training/inference
+on different models. Being able to switch between what engines to use when on what models gives the benefits
+of being able to utilize the strengths of each engine.
+
+## Warning
+Loading multiple deep learning engines will cause OpenMP to load multiple times, which may cause a slowdown
+or memory errors to occur. [Here](#performance-numbers) are the results of a few simple benchmarks that we ran
+
+## Setup
+
+Run the following command to build the project:
+
+`./gradlew build`
+
+## Run the MultiEngine Program
+
+Run the following command to run the project:
+
+`./gradlew run -Dmain=com.examples.MultiEngine`
+
+This will load the Resnet50 models from the respective engine's model zoo and do inference on it with the given image.
+All of this is done within one Java process.
+
+
+## Gradle File Change for MultiEngines
+To have DJL load in multiple engines, just change the build.gradle file to include the engine version snapshot
+wanted. This is a change from the original if, else if, else, loads in the original master gradle file.
```

Comment:
We should aim this guide at users who don't look at our gradle file all of the time. Don't mention the if, else if, else loads since they could confuse users who don't know what you are referring to.

You should also include the Engine implementations as well as the native libraries below. Maybe even stick in the main DJL API too to give it a sense of completeness.

Reply:
I think you mean to show all the dependencies in the gradle file?